src/Paket.Core/Installation/ScriptGeneration.fs

```diff
@@ -1,467 +1,413 @@
-﻿namespace Paket.LoadingScripts
+﻿module Paket.LoadingScripts
 
 open System
 open System.IO
 open Paket
 open Paket.Domain
-open Paket.Queries
 open Mono.Cecil
 
+
+
+
 module PackageAndAssemblyResolution =
+
     let getLeafPackagesGeneric getPackageName getDependencies (knownPackages:Set<_>) openList =
+        
         let leafPackages =
-          openList 
-          |> List.filter (fun p ->
-              not (knownPackages.Contains(getPackageName p)) &&
-              getDependencies p |> Seq.forall (knownPackages.Contains))
+            openList |> List.filter (fun p ->
+                not (knownPackages.Contains(getPackageName p)) &&
+                getDependencies p |> Seq.forall (knownPackages.Contains)
+            )
```

Comment:
For some reason I like the code-style above more. But I'm not a code-style fanatic so I'm fine with this.